src/Symfony/Component/HttpFoundation/Session/Storage/Proxy/SessionHandlerProxy.php

```diff
@@ -12,8 +12,6 @@
 namespace Symfony\Component\HttpFoundation\Session\Storage\Proxy;
 
 /**
- * SessionHandler proxy.
- *
  * @author Drak <drak@zikula.org>
  */
 class SessionHandlerProxy extends AbstractProxy implements \SessionHandlerInterface
```

Comment:
I will also move this class from the `Proxy` folder to `Handler`. What do you think, @nicolas-grekas ?

Reply:
That would need deprecating this class in favor of the new one, and allow triggering a deprecation in AbstractProxy also. LGTM.

Reply:
I think the `SessionHandlerProxy` can be deprecated as well. When we only support `\SessionHandlerInterface` anyway, the proxy does not provide any value anymore AFAIK. I was meant as a way to make `\SessionHandlerInterface` and others behave the same.

Reply:
I was thinking the same. The extra methods that `SessionHandlerProxy` is having can easily be moved to `NativeSessionStorage` as they are relevant only for a native session storage.